content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,141 @@
-  inventory adjustments for new warehouses since they have no existing inventory
-- :guilabel:`Accounting Date`: the date used by accounting teams for bookkeeping related to the
-  inventory
-- :guilabel:`Company`: the company that owns the inventory; can be set as the user's company or as a
-  customer or vendor
-- :guilabel:`Counted Quantities`: choose whether the counted quantities for products being added
-  should default to stock on hand or zero; does not affect inventory adjustments for new warehouses
-  since they have no existing inventory
-
-.. image:: create_a_second_warehouse/inventory-adjustment-configuration.png
-   :align: center
-   :alt: A filled out form for an inventory adjustment.
+If a new warehouse is created to represent a real-world storage location with existing stock, update
+the warehouse's stock using an inventory adjustment.
+
+.. seealso::
+   :doc:`count_products`
 
-Once the form is properly configured, click on :guilabel:`Start Inventory` to be taken to the next
-page where products can be added to the inventory adjustment. Add a new product by clicking on
-:guilabel:`Create` and then fill out the product line as follows:
+Inventory can be added to a new warehouse via an inventory adjustment. Navigate to
+:menuselection:`Inventory app --> Operations --> Physical Inventory`, and click :guilabel:`New`.
+Doing so adds a new line at the bottom of the list of inventory adjustments.
 
-- :guilabel:`Product`: the product being added to inventory
-- :guilabel:`Location`: the location where the product is currently stored in the new warehouse;
-  this can be set as the overall warehouse or a location within the warehouse
+Configure the line by filling out the following fields as necessary:
+
+- :guilabel:`Location`: the location where the product is currently stored in the new warehouse.
+  This can be set as the overall warehouse, or a location within the warehouse.
+- :guilabel:`Product`: the product being added to inventory.
 - :guilabel:`Lot/Serial Number`: the lot that the product belongs to or the serial number used to
-  identify it
-- :guilabel:`On Hand`: the total quantity of the product stored in the location for which inventory
-  is being adjusted; this should be zero for a new location or warehouse
-- :guilabel:`Counted`: the amount of the product that is being added to inventory
-- :guilabel:`Difference`: the difference between the *On Hand* and *Counted* values; this will
-  automatically update to reflect the value entered in the :guilabel:`Counted` column
-- :guilabel:`UoM`: the unit of measure used for counting the product
-
-.. image:: create_a_second_warehouse/product-line-configuration.png
+  identify it, if the product is tracked.
+- :guilabel:`On Hand Quantity`: the total quantity of the product stored in the location for which
+  inventory is being adjusted. For a new warehouse or location, this should be set to `0.00`.
```

Comment:
Consider adding more context around the conditions for the value we're suggesting.
 
It's possible that someone IRL did a physical count at the new warehouse (or new ahead of time), and inputted it into the new warehouse in Odoo before we (i.e., the office admin) performed an "official" batch inventory adjustment.

```suggestion
  inventory is being adjusted. For a new warehouse or location where a count or a prior inventory
  adjustment has not taken place, this should be set to `0.00`.
```